### PR TITLE
Amount input field is highlighted red despite valid amount is entered #19372

### LINKED ISSUE #19372: Amount input field is highlighted red despite valid amount is entered
This issue rarely occurs on real devices but is reproducible in Android Studio emulators. I hope fixing it in Android Studio will resolve it for real devices too.

### Steps:
1. Recover user with some assets.
2. Go to the sent assets page.
3. Enter an available amount.

### Actual result:
The whole input field is highlighted in red.

https://github.com/status-im/status-mobile/assets/52490791/cd3f6207-ae51-4f69-9b65-43d3a8b17a4b

### Expected result:
The input field should only be highlighted in red if the user enters an amount that exceeds what they own.

### Devices:
Android Studio Pixel XL API 33

### PR DESCRIPTION
**Pull request by mmilad75**
fixes #19372 

The issue was that the operator `>` was expecting number but in some cases, a string was passed to it.

### CODE REVIEW

**Review by ulisesmac — APPROVED** [review]
Cool  :+1:

File: src/status_im/contexts/wallet/send/input_amount/view.cljs [hunk 1]
```diff
@@ -86,7 +86,7 @@
 (defn- reset-input-error
   [new-value prev-value input-error]
   (reset! input-error
-    (> new-value prev-value)))
+    (> (js/parseFloat new-value) (js/parseFloat prev-value))))
```
Comment (briansztamfater) [inline src/status_im/contexts/wallet/send/input_amount/view.cljs]: we should use BigNumbers instead of js floats

Comment (mmilad75) [reply src/status_im/contexts/wallet/send/input_amount/view.cljs]: Done

**Review by briansztamfater — APPROVED** [review]

**Review by smohamedjavid — APPROVED** [review]

**Comment by pavloburykh** [comment]
Wlll continue testing once https://github.com/status-im/status-mobile/issues/19729 is fixed (see ISSUE 2 Wrong font colour of amount value)

**Review by ilmotta — APPROVED** [review]

**Comment by pavloburykh** [comment]
Hey @mmilad75. Could you please rebase the PR and resolve conflicts? After that I will continue with testing. Thank you.

**Comment by mmilad75** [comment]
> Hey @mmilad75. Could you please rebase the PR and resolve conflicts? After that I will continue with testing. Thank you.

The issue that was solved here is already solved in another PR. I will close the PR now

**Pull request closed without merging**